docker/image_test.go

```diff
@@ -37,3 +41,15 @@ func TestParseImageFromName(t *testing.T) {
 		}
 	}
 }
+
+func TestParseImageFromRepoAndSha(t *testing.T) {
+	registries := []string{"", "quay.io"}
```

Comment:
`ParseImageFromRepoAndSha` could be passed `index.docker.io` as a registry, too, right?  Do we want to test that case? (covering logic that converts the actual string to `""`)